melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/PDLSystemAuthFilter.java

```diff
@@ -14,23 +14,23 @@
 @Component
 public class PDLSystemAuthFilter implements ExchangeFilterFunction {
 
-    private final RestStsService restStsService;
+    private final RestSts restSts;
```

Comment:
Hva står "Sts" for?

Reply:
secure token service

Reply:
https://en.wikipedia.org/wiki/Security_token_service